packages/paste-core/layout/stack/stories/index.stories.tsx

```diff
@@ -10,95 +9,118 @@ import {Stack, StackOrientation} from '../src';
 const orientationOptions = ['horizontal', 'vertical'];
 const spaceOptions = Object.keys(DefaultTheme.space);
 
-storiesOf('Layout|Stack', module)
-  .addDecorator(withKnobs)
-  .add('Stack', () => {
-    const orientaionValue = select('orientation', orientationOptions, 'vertical') as StackOrientation;
-    const spaceValue = select('spacing', spaceOptions, 'space40') as keyof ThemeShape['space'];
-    return (
-      <Stack orientation={orientaionValue} spacing={spaceValue}>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            First Card
-          </Heading>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Second Card
-          </Heading>
-          <Paragraph>Card content should go here.</Paragraph>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Third Card
-          </Heading>
-        </Card>
-      </Stack>
-    );
-  })
-  .add('Stack - Vertical', () => {
-    return (
-      <Stack orientation="vertical" spacing="space40">
-        <Card>
-          <Heading as="h2" variant="heading20">
-            First Card
-          </Heading>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Second Card
-          </Heading>
-          <Paragraph>Card content should go here.</Paragraph>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Third Card
-          </Heading>
-        </Card>
-      </Stack>
-    );
-  })
-  .add('Stack - Horizontal', () => {
-    return (
-      <Stack orientation="horizontal" spacing="space40">
-        <Card>
-          <Heading as="h2" variant="heading20">
-            First Card
-          </Heading>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Second Card
-          </Heading>
-          <Paragraph>Card content should go here.</Paragraph>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Third Card
-          </Heading>
-        </Card>
-      </Stack>
-    );
-  })
-  .add('Stack - Responsive', () => {
-    return (
-      <Stack orientation={['vertical', 'horizontal', 'vertical']} spacing="space40">
-        <Card>
-          <Heading as="h2" variant="heading20">
-            First Card
-          </Heading>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Second Card
-          </Heading>
-          <Paragraph>Card content should go here.</Paragraph>
-        </Card>
-        <Card>
-          <Heading as="h2" variant="heading20">
-            Third Card
-          </Heading>
-        </Card>
-      </Stack>
-    );
-  });
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Layout/Stack',
+  decorators: [withKnobs],
+};
+
+export const DefaultStack = (): React.ReactNode => {
```

Comment:
Should we be more deliberate about naming these exports? I noticed this one named `DefaultStack` and others just named `Default`.